stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-11.6
+resolver: lts-11.14
```

Comment:
Is it strictly necessary to bump the LTS version here? I tend to avoid such updates in unrelated patches.

Reply:
Now use an extra-dep instead. (`ansi-terminal-0.8.0.4` brings the mintty fix to its `hSupportsANSI`.)